.github/workflows/ocl-sub-module.yml

```diff
@@ -0,0 +1,28 @@
+name: OCL Subscription Module
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  repository_dispatch:
+    types: [qa]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Using Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Installing dependencies
+        run: npm install
+      - name: Run ocl subscription module
+        run: npm run refappOCLSubscriptionModule
```

Comment:
i think **npm run refappOclSubscriptionModule** , here we could also follow bit of openmrs convention.

Reply:
hmmm, what convention 🤔

Reply:
Hey @hadijahkyampeire  is this working locally, can we sny at 5pm today and look through it together.

Reply:
Hey @sherrif10, sure we can sync it is even working here on github.